### DISCUSSION TITLE
Lithium + Psychedelics

### DISCUSSION THREAD

**Discussion by zelixir25** [TripSit/drugs#77] [Drug Interactions]
Evidence suggests that taking psychedelics in combination with lithium can potentially cause seizures. These combinations may also lead to overall unpleasant experiences.

Not sure how we want to handle case studies but one here indicates that lithium may have played a role in an overdose death on 25i-NBOMe.

How do we want to classify these combinations?

lithium + LSD/psilocybin (physical effects)
Nayak, S. M., Gukasyan, N., Barrett, F. S., Erowid, E., Erowid, F., & Griffiths, R. R. (2021). Classic Psychedelic Coadministration with Lithium, but Not Lamotrigine, is Associated with Seizures: An Analysis of Online Psychedelic Experience Reports. Pharmacopsychiatry, 54(5), 240–245. https://doi.org/10.1055/a-1524-2794

Edwards, C., Van Gerpen, S., & Anand, V. (2024). New-Onset Seizures in an Adolescent Following Use of LSD while on Low-Dose Lithium Therapy: A Case Study. South Dakota Medicine: The Journal of the South Dakota State Medical Association, 77(1), 31–35. 

lithium + LSD/psilocybin/phenethylamines (psychological effects)
Simonsson, O., Hendricks, P. S., Chambers, R., Osika, W., & Goldberg, S. B. (2023). Prevalence and associations of challenging, difficult or distressing experiences using classic psychedelics. Journal of Affective Disorders, 326, 105–110. https://doi.org/10.1016/j.jad.2023.01.073

lithium + 25i-NBOMe
Umemura, Y., Andrew, T. A., Jacobs, V. L., Giustini, A. J., Lewis, L. D., & Filiano, J. J. (2015). Fatal 251-NBOMe Intoxication: A New Recreational Risk. Academic Forensic Pathology, 5(1), 91–97. https://doi.org/10.23907/2015.009

**Comment by utaninja** [comment]
Currently, Lithium has been done for LSD, Mushrooms, Mescaline, DOx, NBOMes, 2C-X, 2C-T-X. It has not been done for 5-MEO-xxT 

We have included in our sources that you mentioned above for these combinations-

Nayak, S. M., Gukasyan, N., Barrett, F. S., Erowid, E., Erowid, F., & Griffiths, R. R. (2021). Classic Psychedelic Coadministration with Lithium, but Not Lamotrigine, is Associated with Seizures: An Analysis of Online Psychedelic Experience Reports. Pharmacopsychiatry, 54(5), 240–245. https://doi.org/10.1055/a-1524-2794" currently in the chart right now for these combinations

Simonsson, O., Hendricks, P. S., Chambers, R., Osika, W., & Goldberg, S. B. (2023). Prevalence and associations of challenging, difficult or distressing experiences using classic psychedelics. Journal of Affective Disorders, 326, 105–110. https://doi.org/10.1016/j.jad.2023.01.073


This source would be helpful to add to LSD.

Edwards, C., Van Gerpen, S., & Anand, V. (2024). New-Onset Seizures in an Adolescent Following Use of LSD while on Low-Dose Lithium Therapy: A Case Study. South Dakota Medicine: The Journal of the South Dakota State Medical Association, 77(1), 31–35.

And this source for 25i-NBOMe
Umemura, Y., Andrew, T. A., Jacobs, V. L., Giustini, A. J., Lewis, L. D., & Filiano, J. J. (2015). Fatal 251-NBOMe Intoxication: A New Recreational Risk. Academic Forensic Pathology, 5(1), 91–97. https://doi.org/10.23907/2015.009

Do you think that the above two sources should be added for the other combinations as well?

**Reply by zelixir25** [reply]
I'd be hesitant to add those for other combos especially since they're case studies which are already kinda iffy imo

**Reply by utaninja** [reply]
We can add them to the drugs specific then, should be good!

**Comment by utaninja** [comment]
Are you able to link "Edwards, C., Van Gerpen, S., & Anand, V. (2024). New-Onset Seizures in an Adolescent Following Use of LSD while on Low-Dose Lithium Therapy: A Case Study. South Dakota Medicine: The Journal of the South Dakota State Medical Association, 77(1), 31–35"?

**Reply by zelixir25** [reply]
pubmed doesn't have any abstracts past october 2023. i was able to access it via my school's library subscription. 
https://openurl.ebsco.com/EPDB%3Agcd%3A7%3A9654231/detailv2?sid=ebsco%3Aplink%3Ascholar&id=ebsco%3Agcd%3A174853355&crl=c

**Comment by utaninja** [comment]
I've put 
Edwards, C., Van Gerpen, S., & Anand, V. (2024). New-Onset Seizures in an Adolescent Following Use of LSD while on Low-Dose Lithium Therapy: A Case Study. South Dakota Medicine: The Journal of the South Dakota State Medical Association, 77(1), 31–35.
and
Umemura, Y., Andrew, T. A., Jacobs, V. L., Giustini, A. J., Lewis, L. D., & Filiano, J. J. (2015). Fatal 251-NBOMe Intoxication: A New Recreational Risk. Academic Forensic Pathology, 5(1), 91–97. https://doi.org/10.23907/2015.009
into issues. this should be good!

**Comment by utaninja** [comment]
These have all been pushed. Closing.
